test/canillita_meta_SUITE.erl

```diff
@@ -0,0 +1,18 @@
+-module(canillita_meta_SUITE).
+
+-include_lib("mixer/include/mixer.hrl").
+-mixin([{ktn_meta_SUITE
+        , [ all/0
+           , xref/1
+           , dialyzer/1
+           , elvis/1
+          ]
+        }]).
+
+-export([init_per_suite/1]).
+
+-type config() :: [{atom(), term()}].
+
+-spec init_per_suite(config()) -> config().
+init_per_suite(Config) ->
+    [ {base_dir, "../.."} | Config].
```

Comment:
2 spaces for indentation. And I would try with `{application, canillita}` instead of `{base_dir, "../.."}`. It should work the same and it will produce less confusion on our blog readers once this is published :P